django/santropolFeast/member/tests.py

```diff
@@ -1515,18 +1515,19 @@ def test_view_client_status_update_empty_dates(self):
         )
         self.client.login(username=admin.username, password='test1234')
         data = {
-            'pk': self.active_client.id,
+            'client': self.active_client.id,
+            'status_from': self.active_client.status,
             'status_to': Client.PAUSED,
             'reason': '',
         }
         response = self.client.post(
-            reverse_lazy('member:clientStatusAlterOrScheduleProc',
+            reverse_lazy('member:clientStatusScheduler',
                          kwargs={'pk': self.active_client.id}),
             data,
+            HTTP_X_REQUESTED_WITH='XMLHttpRequest',
             follow=True
         )
-        client = Client.objects.get(pk=self.active_client.id)
-        self.assertEqual(client.status, Client.PAUSED)
+        self.assertTrue(b'This field is required' in response.content)
```

Comment:
It was previously assumed that no dates selected means right now, but it is more consistent this way.